README.md

```diff
@@ -431,4 +431,4 @@ links to the four raw transactions published in the example:
 | B's Decred redemption | [49245425967b7e39c1eb27d261c7fe972675cccacff19ae9cc21f434ccddd986](https://mainnet.decred.org/tx/49245425967b7e39c1eb27d261c7fe972675cccacff19ae9cc21f434ccddd986) |
 ## License
 
-These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
\ No newline at end of file
+These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
```

Comment:
what changed here?

Reply:
I didn't change anything on that line. I just used the github web interface to change the typo and then submitted a PR.

Reply:
Not sure.  I didn't change that line. I only changed the typo using githubs web interface while I was reading it.

Reply:
`git diff` shows
```
-These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
\ No newline at end of file
+These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
```